src/index.js

```diff
@@ -19,3 +19,9 @@ import {Parser} from "./parser";
 export default function parse(code) {
   return new Parser(code).parseScript();
 }
+
+export function parseWithLocation(code) {
```

Comment:
This should probably not be a separate interface, and instead be an optional second parameter to `parse`.